operations/mimir-mixin/dashboards/scaling.libsonnet

```
@@ -11,9 +11,10 @@ local utils = import 'mixin-utils/utils.libsonnet';
         title: '',
         options: {
           content: |||
-            This dashboards shows any services which are not scaled correctly.
-            The table below gives the required number of replicas and the reason why.
-            We only show services without enough replicas.
+            This dashboard can help identify scaling-related issues by suggesting services which may benefit from scaling up.
+            The table below gives a suggested number of replicas and the reason why.
```

```suggestion
            The table that follows contains a suggested number of replicas and the reason why.
```
It is considered best practice to remove "below" or "above" in documentation when referring to where text appears. This could be "left" or "right" on another display device other than a monitor.